### PR TITLE
Math magicians - Components

### PR DESCRIPTION
**Pull request by jcmunav63**
**Description**
In this project, you should continue with the development of the Math Magicians app. You will develop a React component that will hold the core functionality: a calculator.

**General requirements**
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that you used the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

**JavaScript requirements**
- Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

**Project requirements**
In the previous project, you worked on the setup of a React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html), now you should:
- Delete all the boilerplate from Create React App (text, images, styles).
- Create a new directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:
            image
- Implement the design only (JSX and CSS)
   - Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use functional components
- Create at least 1 child component
- Once the Calculator is ready, import it and use it in your top-level component: App.js
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <span className="key1">9</span>
+      <span className="key1b">*</span>
+      <span className="key1">4</span>
+      <span className="key1">5</span>
+      <span className="key1">6</span>
+      <span className="key1b">-</span>
+      <span className="key1">1</span>
+      <span className="key1">2</span>
+      <span className="key1">3</span>
+      <span className="key1b">+</span>
```
Comment (codepantha) [inline src/components/Calculator.js]: - [ ] According to the project's requirements, the `<Calculator />` component should have at least one child component. You may create a reusable `<Button />` component for the buttons.
- [ ] Please let's use semantic HTML. Calculators have `buttons`, so kindly replace the `<span>` with `<button>` as it improves accessibility, readability, and maintainability,

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      <span className="keys2">0</span>
+      <span className="key1">.</span>
+      <span className="key1">=</span>
+    </div>
```
Comment (codepantha) [inline src/components/Calculator.js]: - [ ] Please ensure that the background color of the "=" button is the same as that of the other operators just like in the original design. Please see the annotated diagram below for more information 👇 
### Original Design
image
### Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @jcmunav63 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Boilerplate code deleted 📌 
✔️ Created `<Calculator />` component inside `/components` directory 📌 
✔️ No linter errors 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @jcmunav63,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**